modules/kallistobustools/count/main.nf

```diff
@@ -36,12 +38,11 @@ process KALLISTOBUSTOOLS_COUNT {
         -g $t2g \\
         $cdna \\
         $introns \\
-        --workflow $workflow_mode \\
+        --workflow $sc_workflow \\
```

Comment:
also, can this `--workflow $sc_worfklow` be passed as args as well?
Sorry, I'm not at all familiar with the tool and params

Reply:
Not sure either

Reply:
@fmalmeida, I saw you re-added this. 

We were wondering if this arguments can be passed via `ext.args`. But happy to keep `--workflow` if you think it makes more sense!

Reply:
Sorry for the noise then 🙂

Reply:
But to use as `ext.args` we would need to also change the `REF` module, right?

Reply:
Would be ideal if both are working in the same way as  they belong together yeah

Reply:
So, that being said. Maybe we should stick with the variable then, because the `REF` module seems to dependent of it and it would be difficult to extract the fact that the standard module is being selecte through `ext.args` to then select the appropriate command line.

Look how this is in `REF`:
https://github.com/nf-core/modules/blob/b33a59e756ede6107c9a457d5a5cddb63573487e/modules/kallistobustools/ref/main.nf#L27-L64

Reply:
Agreed, lets keep it like this

Reply:
Just to check.
You left `COUNT` as not using the variable and `REF` using? Or you re-added the variable to `COUNT` so they are the same?